bzt/modules/selenium.py

```diff
@@ -123,19 +117,31 @@ def prepare(self):
             runner_config['props-file'] = self.engine.create_artifact("customrunner", ".properties")
 
         runner_config["script-type"] = script_type
-        runner_working_dir = self.engine.create_artifact(runner_config.get("working-dir", "classes"), "")
-        runner_config["working-dir"] = runner_working_dir
+        runner_config["working-dir"] = working_dir
         runner_config.get("artifacts-dir", self.engine.artifacts_dir)
-        runner_config.get("report-file", self.kpi_file)
-        runner_config.get("err-file", self.err_jtl)
```

Comment:
you stopped using the field, but left the unused field to exist?
Anyway, keeping these public fields is good practice, other modules might want to get access to them.